imageio/plugins/rawpy.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# imageio is distributed under the terms of the (new) BSD License.
+
+""" Read/Write images using rawpy.
+
+rawpy is an easy-to-use Python wrapper for the LibRaw library. 
+It also contains some extra functionality for finding and repairing hot/dead pixels.
+"""
+
+import rawpy
+import numpy as np
+
+from ..core.request import URI_BYTES, InitializationError, IOMode, Request
+from ..core.v3_plugin_api import PluginV3
+
+
+class RawPyPlugin(PluginV3):
+    """A class representing the rawpy plugin.
+    This class is a subclass of PluginV3.
+
+    Methods
+    -------
+
+    .. autosummary::
+    :toctree: _plugins/rawpy
+
+    RawPyPlugin.read
+    """
+    def __init__(self, request: Request) -> None:
+        """Instantiates a new rawpy plugin object
+
+        parameters
+        ----------
+        request: {Request}
```

Comment:
```suggestion
        request : Request
```

Our docs are a bit inconsistent on this end (due to legacy). I'm trying to slowly move towards a common format.

Reply:
Thanks for the suggestion. I have updated the script accordingly.